### DISCUSSION TITLE
Is there a way to stop costant requests on /admin/api/notifications?

### DISCUSSION THREAD

**Discussion by ValerianMemsk** [orchidsoftware/platform#2484] [Q&A] [answered]
I do not need notifications in my application, so in platform.php config file i set 
```
'notifications' => [
        'enabled'  => false,
        'interval' => 0,
],
```
It did stop requests on /admin/api/notifications, but somehow only in my local development. After I pushed changes to my test server with "production" env adr reloaded php there is still those requests send constantly. Can anyone tell how can I fix it? Its not really a problematic but rather annoying? because i want to monitor requests and revlevant once lost in the sea of this "parasitic" ones

**Comment by tabuna** [comment, marked as answer]
Make sure the configuration cache has been cleared or updated
